src/suggest/date-suggest.ts

```diff
@@ -9,12 +9,11 @@ import {
   TFile,
 } from "obsidian";
 import type NaturalLanguageDates from "src/main";
+import t from "../lang/helper";
+
 
-interface IDateCompletion {
```

Comment:
Since this only had a `label: string` I removed it, that made the code a bit more readable (the interface contained only a string, now it's just a string), but if it was necessary then I can of course revert it